### PR TITLE
Update IConfiguration.xml

### PR DESCRIPTION
**Pull request by electricalArt**
Add a additional remark paragraph

### CODE REVIEW

File: xml/Microsoft.Extensions.Configuration/IConfiguration.xml [hunk 1]
```diff
@@ -114,7 +114,9 @@
         <summary>Gets a configuration sub-section with the specified key.</summary>
         <returns>The <see cref="T:Microsoft.Extensions.Configuration.IConfigurationSection" />.</returns>
         <remarks>This method will never return <see langword="null" />. If no matching sub-section is found with the specified key,
-                an empty <see cref="T:Microsoft.Extensions.Configuration.IConfigurationSection" /> will be returned.</remarks>
+                an empty <see cref="T:Microsoft.Extensions.Configuration.IConfigurationSection" /> will be returned.
+        
+                You can reach sub-subsection by passing "section1:section2:section.." as key parameter</remarks>
```
Comment (eerhardt) [inline xml/Microsoft.Extensions.Configuration/IConfiguration.xml:119]: I think the grammar may be off a bit here.

```suggestion
                You can reach a sub-subsection by passing "section1:section2:section.." for <paramref name="key"/>.</remarks>
```

?

Comment (Narick9) [reply xml/Microsoft.Extensions.Configuration/IConfiguration.xml]: Yes, this is right, thank you

**Review by eerhardt — APPROVED** [review]
LGTM. Thanks!

**Comment by eerhardt** [comment]
cc @maryamariyan @safern @tarekgh

**Review by tarekgh — APPROVED** [review]

**Review by safern — APPROVED** [review]

**Pull request merged**